lib/app/controllers/inline_forms_controller.rb

```diff
@@ -271,4 +284,7 @@ def unravel(deep_hash, level=-1)
     a
   end
 
+  def revert_params
+    params.require(:id).permit(:update, :comchannel)
```

Comment:
Yeah, this was just put in place to not "break" certain things, still doing some cleanup work.